provider/runbook_resource_test.go

```diff
@@ -83,6 +84,10 @@ func TestAccRunbookResource_update(t *testing.T) {
 						"firehydrant_runbook.test_runbook", "steps.#", "1"),
 					resource.TestCheckResourceAttr(
 						"firehydrant_runbook.test_runbook", "steps.0.name", "Notify Channel"),
+					resource.TestCheckResourceAttrSet(
+						"firehydrant_runbook.test_runbook", "steps.0.repeats"),
```

Comment:
can we check that the repeats value is set to true or false in basic and update?
```suggestion
					resource.TestCheckResourceAttr(
						"firehydrant_runbook.test_runbook", "steps.0.repeats", "true"),
```

i'd also like to add this to the `testAccCheckRunbookResourceExistsWithAttributes_basic` and `testAccCheckRunbookResourceExistsWithAttributes_update` tests

Reply:
ah thats what I was doing wrong, I was trying to do just `true` and I guess it requires a string.